### PR TITLE
Display Experiment results

### PR DESCRIPTION
**Pull request by george-mzai**
## What's changing

- Table display for experiment results (different design from the previous tables)
- Click to copy experiment ID
- Highlight the selected item in both experiments' and datasets' tables
- Prevent user to create an experiment with datasets without ground_truth
- Allow streaming logs when an experiment is in progress

## How to test it

1. Upload a dataset without G_T
2. Try to trigger an experiment with that dataset( popup table menu, button inside dataset details, dropdown select in exepriment form)
3. Upload a dataset with G_T
4. Trigger an experiment
5. Observe status update both on experiments' table and experiment's details on the right side panel.
6. On experiment details copy the experiment id , use it in the openAPI for retrieving information(eg logs)
7. Open _View Results_ when experiment is finished

## Additional notes for reviewers
Try to navigate around while the experiment is running.

## I already...
Started working on notification service which will be available on the upcoming PR

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ services:
       - LOCALSTACK_AUTH_TOKEN=${LOCALSTACK_AUTH_TOKEN}
       - PERSISTENCE=1
       - SNAPSHOT_SAVE_STRATEGY=ON_REQUEST
+      - EXTRA_CORS_ALLOWED_ORIGINS=*
```
Comment (peteski22) [inline docker-compose.yaml:19]: This is required as currently, we return a URL to the calling client which points to the local stack resource.

File: .github/workflows/pull_request_do_not_merge.yaml
Comment (peteski22) [inline .github/workflows/pull_request_do_not_merge.yaml:1]: From: https://github.com/mozilla-ai/lumigator/commit/fbb3fec561f2f041e483e74e63f45e2082a010a6

File: lumigator/frontend/Dockerfile
Comment (peteski22) [inline lumigator/frontend/Dockerfile:1]: From: https://github.com/mozilla-ai/lumigator/commit/6dd6eb70bc66ff58633e853ee453032a41f5b78b#diff-f0231c6f17104918d8b21fbbcb345e53edaea7fe385f79781d871729119de4f3R20

File: lumigator/frontend/src/services/experiments/experimentService.js [hunk 2]
```diff
@@ -28,14 +29,35 @@ async function triggerExperiment(experimentPayload) {
         'Content-Type': 'application/json',
       },
     });
+    // console.log(response);
```
Comment (peteski22) [inline lumigator/frontend/src/services/experiments/experimentService.js:32]:
```suggestion
```

File: lumigator/frontend/src/services/experiments/api.js [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
-export const PATH_EXPERIMENTS_ROOT = () => `health/jobs/`
-export const PATH_EXPERIMENT_DETAILS = (experiment_id) => `health/jobs/${experiment_id}`
+export const PATH_EXPERIMENTS_ROOT = () => `health/jobs/`;
+export const PATH_EXPERIMENT_DETAILS = (experiment_id) => `health/jobs/${experiment_id}`;
+export const PATH_EXPERIMENT_LOGS = (id) => `health/jobs/${id}/logs`
 // these 👆 endopoints do not belong to the experiments API,
+// temporary solution until
+// migration to "job" API route
+export const PATH_EXPERIMENT_RESULTS = (job_id) => `jobs/${job_id}/result/download`
+export const PATH_EXPERIMENTS_EVALUATE = () => `jobs/evaluate/`
```
Comment (ividal) [inline lumigator/frontend/src/services/experiments/api.js:8]: Just a quick note that the preferred interface (and the one that should remain stable for the MVP and will handle the required combinations of jobs under the hood) is `/experiments`. 

Using `/jobs` forces the UI to make workflow decisions (the temporary need to do so was fixed with #377). 

Give it a spin and shout if there is an issue! :)

@agpituk

Comment (george-mzai) [reply lumigator/frontend/src/services/experiments/api.js]: Results and evaluation(triggering experiment) were update.
Remaining issues are: 

1. `/experiments` doesn't provide the **RUNNING** status
2. `/experiments/{experiment_id}` is missing
     🔎 Used dataset
     🔎 Tested model
     🔎 Use case
     🔎 Number of samples

3. `/experiments/{experiment_id}/results` returns 404

**Review by peteski22 — APPROVED** [review]

**Pull request merged**